octokit/pagination.py

```diff
@@ -0,0 +1,38 @@
+from .resources import Resource
+
+class Pagination(object):
+  def __init__(self, *args, **kwargs):
+    # TODO (howei): possibly extract auto_paginate from kwargs
+    # so users can do client = Client(auto_paginate=True)
+    self.auto_paginate = False
+    self.last_response = None
```

Comment:
Actually I see that only the rate limit response callback is setting the last response, but we want to keep track of the last response at all times not just when rate limit fetches the rate limit end point?